binding.gyp

```diff
@@ -65,7 +65,28 @@
           "msvs_settings": {
             "VCCLCompilerTool": {
               "ExceptionHandling": 1,  # /EHsc
-              "AdditionalOptions": ['-std:c++17']
+              "AdditionalOptions": [
+                "-std:c++17",
+                "/guard:cf",
+                "/w34244",
+                "/w34267",
+                "/sdl",
+                "/ZH:SHA_256"
+              ]
+            },
+            "VCLinkerTool": {
+              "AdditionalOptions": [
+                "/guard:cf",
+              ]
+            }
+          }
+        }],
+        ['OS=="win" and target_arch=="x64"', {
+          "msvs_settings": {
+            "VCLinkerTool": {
+              "AdditionalOptions": [
+                "/CETCOMPAT",
```

Comment:
V8 doesn't support CET yet https://source.chromium.org/chromium/chromium/src/+/main:v8/gni/v8.gni;l=179-181, addon cannot be marked compatible.